performance_test/src/experiment_execution/analysis_result.hpp

```diff
@@ -158,6 +161,7 @@ class AnalysisResult
 #pragma db transient
 #endif
   rusage m_sys_usage;
+  const CpuInfo m_cpu_info = {};
```

Comment:
`= {}` unnecessary as it has a constructor.

Reply:
Removed